app/models/recipient.rb

```diff
@@ -27,4 +27,8 @@ def self.exportable_attributes
   def self.from_approved_users
     joins(:created_by_user).where.not(users: { approved_at: nil })
   end
+
+  def self.visible_to_user(user)
+    from_approved_users.where(mobile_network_id: user.mobile_network_id)
+  end
```

Comment:
I think it'd be clearer if this method was:
```suggestion
  def self.on_mobile_network(mobile_network:)
    from_approved_users.where(mobile_network_id: mobile_network.id)
  end
```
so on the caller side it'd be `Recipient.on_mobile_network(@user.mobile_network)`, which would make it easier to grok first time, IMHO.

Reply:
I see your point, but there's also the limitiation to approved-only - how about just:
```
def self.on_mobile_network(mobile_network)
   where(mobile_network_id: mobile_network.id)
 end
```

so then the call becomes:

`Recipient.from_approved_users.on_mobile_network(@user.mobile_network)`

?

Reply:
sounds good, :shipit: